1-hello-world/app.js

```diff
@@ -27,7 +27,7 @@ app.get('/', (req, res) => {
 if (module === require.main) {
   // [START server]
   // Start the server
-  const server = app.listen(process.env.PORT || 8080, () => {
+  const server = app.listen(process.env.PORT || 8081, () => {
```

Comment:
I don't think the right solution is to manually hard code each step of the tutorial to a different port for the sake of the tests. Regardless of which step a user is working on, the app should listen on 8080 without any intervention from the user.